submissions/Harnytskyi/js-post-oop/index.js

```diff
@@ -7,28 +7,23 @@
 
 // ======== OBJECTS DEFINITIONS ========
 class Inhabitant {
-    constructor(species, name, gender, saying, friends = []) {
+    constructor(species, name, gender, saying) {
         this.species = species;
         this.name = name;
         this.gender = gender;
         this.saying = saying;
-        this.friends = friends;
+        this.addFriends = function (friends){
+            this.friends = friends;
+        };
```

Comment:
> Just **use** it the way the method should be used per your design, just within the constructor.

Note verb **use**. Not **define**.

Bring `addFriends` method definition back. Bring `friends` argument back to the constructor signature - why wouldn't we allow assign friends at construction time?

Use the `addFriends` method to initialize friends in constructor.

Feel free asking questions.

Reply:
Do you mean, when creating an instance, "friends" are assigned?
And creating an instance would look like this:
`const man = new Man('John', 'Hello, everybody!', [woman, dog]);` ?

Reply:
The way you use `addFriends` directly in your code is as follows:
```
man.addFriends([woman, dog]);
woman.addFriends([cat]);
dog.addFriends([man]);
```

Reply:
Both options are basically valid.
One can add friends (those that already exist) at a new object instantiation. And then at any time after.

Reply:
BTW, current implementation is rather `assignFriends` as it won't add friends at the next call but rather replace them.